benchmark/opperf/utils/benchmark_utils.py

```diff
@@ -67,28 +61,17 @@ def _run_nd_operator_performance_test(op, inputs, run_backward, warmup, runs, ar
         raise ValueError("Incorrect input for profiler. Valid input - 'python' or 'native'")
 
     # Warm up, ignore the profiler output
-    if not args_list:
-        _, _ = benchmark_helper_func(op, warmup, [], **kwargs_list[0])
-    else:
-        _, _ = benchmark_helper_func(op, warmup, args_list[0], **kwargs_list[0])
+    _, _ = benchmark_helper_func(op, warmup, **kwargs_list[0])
 
     # Run Benchmarks
     op_benchmark_result = {op.__name__: []}
     logging.info("Begin Benchmark - {name}".format(name=op.__name__))
-    if not args_list:
-        for idx, kwargs in enumerate(kwargs_list):
-            _, profiler_output = benchmark_helper_func(op, runs, [], **kwargs)
-
-            # Add inputs used for profiling this operator into result
-            profiler_output["inputs"] = inputs[idx]
-            op_benchmark_result[op.__name__].append(profiler_output)
-    else:
-        for idx, (args, kwargs) in enumerate(zip(args_list, kwargs_list)):
-            _, profiler_output = benchmark_helper_func(op, runs, args, **kwargs)
+    for idx, kwargs in enumerate(kwargs_list):
+        _, profiler_output = benchmark_helper_func(op, runs, [], **kwargs)
```

Comment:
We should incorporate our fix (removing the empty list) here as well.

Reply:
Addressed